.gitlab-ci.yml

```diff
@@ -487,6 +714,7 @@ build_windows_msi_x64:
   variables:
     WINDOWS_BUILDER: 'true'
     CREDENTIALS_FILE_PATH: 'c:\users\gitlab\.aws\config'
+    PYTHON_RUNTIMES: '3'
   tags: ["runner:windows-agent6"]
```

Comment:
Should be `["runner:windows-agent7"]` here.

Reply:
Actually no, this is fine. The `runner:windows-agent6` has everything we need to build. You can check the artifact in any of the successful pipelines.